compiler/rustc_parse/messages.ftl

```diff
@@ -406,7 +406,8 @@ parse_invalid_logical_operator = `{$incorrect}` is not a logical operator
     .use_amp_amp_for_conjunction = use `&&` to perform logical conjunction
     .use_pipe_pipe_for_disjunction = use `||` to perform logical disjunction
 
-parse_invalid_meta_item = expected unsuffixed literal or identifier, found `{$token}`
+parse_invalid_meta_item = expected unsuffixed literal{$or_ident}, found `{$token}`
```

Comment:
I forgot where this is documented, but please avoid using untranslatable strings like this.

Instead you should either have a flag and match it here (I'm not sure what the syntax is) or simply have two error texts — one with and the other without ` or identifier`.